distributed/tensor_parallelism/tensor_parallel_example.py

```diff
@@ -1,11 +1,17 @@
 import argparse
-
+import os
 import torch
-import torch.multiprocessing as mp
 
-from torch.distributed._tensor import DeviceMesh
-from torch.distributed.tensor.parallel import PairwiseParallel, parallelize_module
-from utils import cleanup, setup, ToyModel
+from torch.distributed._tensor.device_mesh import init_device_mesh
+from torch.distributed._tensor import DeviceMesh, DTensor, Replicate, Shard
+
+from torch.distributed.tensor.parallel import (
+    PairwiseParallel,
```

Comment:
ditto: we want to delete PairwiseParallel soon

Reply:
have removed for all 3.  for reference, you don't need to mark each file re: review comments...just show on one and note that it applies to all ;)